feature-libs/checkout/b2b/root/index.ts

```diff
@@ -0,0 +1,8 @@
+/** AUGMENTED MODELS NEEDS TO BE EXPORTED FIRST */
+export * from './augmented-models/index';
```

Comment:
does it have to be first?

Other feature-libs has it before last as the name of the folder was called ./model/index